### DISCUSSION TITLE
KODI Integration

### DISCUSSION THREAD

**Discussion by SecularSteve** [vpnhood/VpnHood#470] [Ideas]
More specifically, how to possibly set up a profile for Zomboided's VPN Manager.

https://github.com/Zomboided/service.vpn.manager

If it cannot be done due to VPNHood's more intricate hybrid design, can an Add-On at least be considered, let alone be possible to make?

**Comment by trudyhood** [comment]
I'm sorry, but it is not on the table right now.
